src/store/editor/editor-generators.ts

```diff
@@ -138,6 +139,35 @@ const normalizeParticipants = (
 		? abstractParticipants.map((abstractParticipant) => normalizeParticipant(abstractParticipant))
 		: [];
 
+const byType =
```

Comment:
IMO this function name is not describing the purpose of the function. This, in the long run, could lead to the need to see the implementation to understand it

Reply:
do you have a better name to suggest? the function creates a predicate on a Participant for a given role, maybe filterParticipantByRole ?

Reply:
since it returns a predicate it could be `hasRole` or `rolePredicate`